spec/models/topic_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  image      :string
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_topics_on_name  (name) UNIQUE
+#
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+  let(:topic) { build(:topic) }
+
+  it 'validates name presence' do
```

Comment:
Maybe we can use https://github.com/thoughtbot/shoulda-matchers#activerecord-matchers 
to validate name uniqueness and presence

Reply:
now I am use the Shoulda gem